app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Typography, Dropdown, Tag, Tooltip, Avatar } from "neetoui";
+
+const Note = ({
+  id,
+  title,
+  description,
+  tag,
+  status,
+  created_at,
+  avatar,
+  setShowDeleteAlert,
+  setDeleteNoteId,
+  setIsOpen,
+}) => {
+  const deleteHelper = () => {
```

Comment:
This is a bad name for a function. This is not a `helper`. You can are handling the deletion of the `note` entity by passing this function to `onClick`. Thus name it appropriately:
```suggestion
const handleDelete = () => {
```

Reply:
This is resolved.